pom.xml

```diff
@@ -69,6 +69,7 @@
         <groovy-cps.version>1.34</groovy-cps.version>
         <node.version>16.17.0</node.version>
         <npm.version>8.18.0</npm.version>
+        <yarn.version>1.22.19</yarn.version>
```

Comment:
Not listed on https://github.com/yarnpkg/berry/releases; is this just the previous hard-coded value or something?

Reply:
This is the last release of the 1.x line. 1.x is unsupported for several years, but using berry (3.x) out of the box requires https://github.com/jenkinsci/plugin-pom/pull/625 on our end first.